### PR TITLE
hw6.2

### PR DESCRIPTION
**Pull request by OlessiaKharchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/Rectangle.java [hunk 1]
```diff
@@ -17,14 +17,45 @@ public Rectangle(Integer width, Integer length, String color) {
     }
 
     public Integer getWidth() {
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Rectangle.java]: we don't need these blank lines

File: src/main/java/core/basesyntax/Rectangle.java [hunk 2]
```diff
@@ -17,14 +17,45 @@ public Rectangle(Integer width, Integer length, String color) {
...
+    public boolean equals(Object obj) {
+        if (obj == this) {
+            return true;
+        }
+        if (obj == null || obj.getClass() != this.getClass()) {
+            return false;
+        }
+        Rectangle rectangle = (Rectangle) obj;
+        return width == rectangle.width || (width != null
+                && width.equals(rectangle.getWidth())
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Rectangle.java]:
```suggestion
        return (width == rectangle.width || (width != null
                && width.equals(rectangle.getWidth()))
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Rectangle.java (hunk 1 above)
Comment (OlessiaKharchenko) [reply src/main/java/core/basesyntax/Rectangle.java]: ok, done!

File: src/main/java/core/basesyntax/Rectangle.java (hunk 2 above)
Comment (OlessiaKharchenko) [reply src/main/java/core/basesyntax/Rectangle.java]: done!

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**